react/Checkbox/Checkbox.test.js

```diff
@@ -1,126 +1,61 @@
-  }
+  const simpleProps = {
+    id: 'still-in-role',
+    label: 'Still in role'
+  };
 
-  function renderToDom(jsx) {
-    element = jsx;
-    checkbox = renderIntoDocument(element);
-    rootElement = scryRenderedDOMComponentsWithClass(checkbox, 'root')[0];
-    label = scryRenderedDOMComponentsWithClass(checkbox, 'label')[0];
-    input = findRenderedDOMComponentWithClass(checkbox, 'input');
-  }
-
-  it('should have a displayName', () => {
-    render(<Checkbox />);
-    expect(element.type.displayName).to.equal('Checkbox');
-  });
-
-  it('should pass through id', () => {
-    const id = 'test';
-    renderToDom(<Checkbox id={id} />);
-    expect(input.id).to.equal(id);
+  it('should render with simple props', () => {
+    const wrapper = shallow(<Checkbox {...simpleProps} />);
+    expect(wrapper).toMatchSnapshot();
   });
 
-  it('should pass through className', () => {
-    const className = 'test';
-    renderToDom(<Checkbox className={className} />);
-    expect(rootElement.classList.contains('test')).to.equal(true);
+  it('should render with className', () => {
+    const className = 'testClassname';
+    const wrapper = shallow(<Checkbox {...simpleProps} className={className} />);
+    expect(wrapper).toMatchSnapshot();
   });
 
-  it('should have `htmlFor` equal to `id` when `label` is specified', () => {
-    render(<Checkbox id="still-in-role" label="Still in role" />);
-    expect(label.props).to.contain.keys({ htmlFor: 'still-in-role' });
+  it('should render with standard checkbox style', () => {
+    const wrapper = shallow(<Checkbox {...simpleProps} type="standard" />);
+    expect(wrapper).toMatchSnapshot();
   });
 
-  describe('when type is not set', () => {
-    it('should render standard checkbox', () => {
-      render(<Checkbox />);
-      expect(standard).to.not.equal(null);
-    });
-  });
-
-  describe('when type is standard', () => {
-    it('should render standard checkbox', () => {
-      render(<Checkbox type="standard" />);
-      expect(standard).to.not.equal(null);
-    });
-  });
-
-  describe('when type is button', () => {
-    it('should render button checkbox', () => {
-      render(<Checkbox type="button" />);
-      expect(button).to.not.equal(null);
-    });
+  it('should render with button style', () => {
+    const wrapper = shallow(<Checkbox {...simpleProps} type="button" />);
+    expect(wrapper).toMatchSnapshot();
   });
 
   describe('inputProps', () => {
     it('should invoke the onChange handler when touched', () => {
-      const handleChange = sinon.spy();
-      handleChange.reset();
+      const handleChange = jest.fn();
       const props = {
-        id: 'still-in-role',
-        label: 'still in role',
+        ...simpleProps,
         inputProps: {
           onChange: handleChange,
           checked: false
         }
       };
-      renderToDom(<Checkbox {...props} />);
+      const check = { target: { checked: true } };
```

Comment:
I would find this test easier to read if this const was called something like `checkedEvent`.  The name seems a bit misleading to me